build-system/global-configs/experiments-config.json

```diff
@@ -1,7 +1,7 @@
 {
   "experimentA": {
     "name": "ANALYTICS_VENDOR_SPLIT",
-    "command": "gulp dist --defineExperimentConstant=ANALYTICS_VENDOR_SPLIT",
```

Comment:
hmmmm we need this `command` to build experiment bundle in release. Can we add --fortesting automatically, or we can introduce another test command? WDYT?

Reply:
ooh I see, undid the change and did this instead
https://github.com/ampproject/amphtml/blob/4de59615a9c50d8f6da1ed0d0878d6771d56fa7a/build-system/pr-check/dist-experiment.js#L42-L45

Reply:
Great! That sounds good!